### PR TITLE
TS-19 FEATURE - NavBar

### PR DESCRIPTION
**Pull request by dcarg**
Have Added the Navbar Component to the top of the page.
The Navbar takes a title param, includes a TS Logo and a link to the root of the site.

I originally had set the background as just black with white text overlayed. Updated to cyan as maybe our theme colour? What do you think? For MVP this'll do and can work on styling later. Eventually I want it to take system colours (dark or light mode)

Have added the initial setup of custom local font. The font is working but haven't quite figured out how to use different variations of the font and such.

I'm not married to this logo just chucking it out as a first attempt.
I generated the logo at logo.com (also tried logoai.com)

Currently I've just hardcoded the logo svg's fill to #FFFFF (white) but in the future would like to make an improvement by making the SVG Icon into a react component so we can pass tailwind classes to style it (change color, size, margin/padding etc)

Came across a pretty powerful tool to do this here which I'd like to follow up in the future:
[https://stackoverflow.com/questions/65676689/next-js-how-can-i-change-the-color-of-svg-in-next-image](url)
[https://transform.tools/](url)

Screenshot 2023-12-07 at 10 08 58 pm

### CODE REVIEW

File: src/app/shared/components/Navbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+interface NavbarProps {
+  title: string,
+}
+
+const NavBar = async (props: NavbarProps) => {
+  const { title } = props
+
+  return (
+    <div className="bg-cyan-700 h-[60px] w-full ">
+      <Link href="/">
```
Comment (dcarg) [inline src/app/shared/components/Navbar/index.tsx]: I've wrapped this Link around both the Logo & the Title. Not sure if we want this OR just wrapping it around the logo only.

File: src/app/layout.tsx [hunk 2]
```diff
@@ -1,12 +1,34 @@
 import type { Metadata } from 'next'
 import { Inter } from 'next/font/google'
+import localFont from "next/font/local"
 import './globals.css'
 
+const futura = localFont(
```
Comment (Mark-Ball) [inline src/app/layout.tsx:16]: can we move this to below the imports?

File: src/app/layout.tsx [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
-import '@fortawesome/fontawesome-svg-core/styles.css';
-import { config } from "@fortawesome/fontawesome-svg-core";
+import '@fortawesome/fontawesome-svg-core/styles.css'
+import { config } from "@fortawesome/fontawesome-svg-core"
 config.autoAddCss = false;
 
+import NavBar from '@components/NavBar'
 import TopBar from '@components/TopBar'
 
 const inter = Inter({ subsets: ['latin'] })
```
Comment (Mark-Ball) [inline src/app/layout.tsx]: will we still use this inter font somewhere? if we're all-in on futura-pt we may as well remove it

File: src/app/shared/components/Navbar/index.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+interface NavbarProps {
+  title: string,
+}
+
+const NavBar = async (props: NavbarProps) => {
+  const { title } = props
+
+  return (
+    <div className="bg-cyan-700 h-[60px] w-full ">
```
Comment (Mark-Ball) [inline src/app/shared/components/Navbar/index.tsx]: i think these hardcoded heights are throwing off our alignment, i think it looks better if we remove the height here and the height and width on line 15

File: src/app/shared/components/Navbar/index.tsx (hunk 1 above)
Comment (Mark-Ball) [reply src/app/shared/components/Navbar/index.tsx]: i would just wrap the Image, currently the whole navbar is clickable which feels a little strange

**Review by Mark-Ball — CHANGES_REQUESTED** [review]

File: src/app/shared/components/Navbar/index.tsx (hunk 1 above)
Comment (dcarg) [reply src/app/shared/components/Navbar/index.tsx]: Agreed & updated

File: src/app/layout.tsx (hunk 3 above)
Comment (dcarg) [reply src/app/layout.tsx]: Well I do want to use two fonts. Futura really just for Headings etc and then a normal font for most regular text, need to play around with it

File: src/app/shared/components/Navbar/index.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+              src="/ts-logo.svg"
+              alt="TeamSheet Logo"
+              priority
+              height={30}
+              width={30}
+            />
+          </div>
+        </Link>
+        
+        <div className="flex align-middle items-center h-full max-w-column w-full text-white text-xl ml-[5px]">
```
Comment (dcarg) [inline src/app/shared/components/Navbar/index.tsx]: Now I can't seem to get this text to align vertically for some reason...

**Review by Mark-Ball — APPROVED** [review]

**Pull request merged**